test/config_parser_test.cc

```diff
@@ -269,7 +271,7 @@ SUITE(configTests) {
 
 GREATEST_MAIN_DEFS();
 
-int main(int argc, char** argv){
+int main(int argc, char** argv) {
```

Comment:
You could add test (and fix code) for invalid HTTP header names (or invalid value), i.e. any string containing a separator like: separators  (c.f. token https://tools.ietf.org/html/rfc2616#section-2.2 )